content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -1,350 +1,482 @@
 
-    private String name;
-    private int age;
-    private String color;
-    private String breed;
+   private String name;
 
+   private int age;
 
-    //Dog arg constructor
-    public Dog(String name, int age, String color, String breed) {
-        this.name = name;
-        this.age = age;
-        this.color = color;
-        this.breed = breed;
-    }
+   private String color;
 
-    public boolean isSleeping(){
-        System.out.println("Sleeping");
-        return true;
-    }
+   private String breed;
 
-    public boolean isEating(){
-        System.out.println("Eating");
-        return true;
-    }
+   //Dog arg constructor
 
-    public boolean isbarking(){
-        System.out.println("Dog barking...");
-        return true;
-    }
+   public Dog(String name, int age, String color, String breed) {
+
+       this.name = name;
+
+       this.age = age;
+
+       this.color = color;
+
+       this.breed = breed;
+
+   }
+
+   public boolean isSleeping(){
+
+       System.out.println("Sleeping");
+
+       return true;
+
+   }
+
+   public boolean isEating(){
+
+       System.out.println("Eating");
+
+       return true;
+
+   }
+
+   public boolean isbarking(){
+
+       System.out.println("Dog barking...");
+
+       return true;
+
+   }
+
+   public boolean isRunning(){
+
+       System.out.println("Dog running");
+
+       return true;    
+
+   }   
 
-    public boolean isRunning(){
-        System.out.println("Dog running");
-        return true;	
-    }   
 }
+
 ```
 
 To create a new dog object
 
 ```java
+
 public class Dog_Main {
-   public static void main(String[] args) {
-       Dog littleDog = new Dog("Bingo", 6,"Brown", "German Shepherd");
-       System.out.println("Is the dog running:? "+ littleDog.isrunning());
 
-       System.out.println();
+  public static void main(String[] args) {
 
-       System.out.println("is the dog sleeping:? " + littleDog.isSleeping());
+      Dog littleDog = new Dog("Bingo", 6,"Brown", "German Shepherd");
 
-       System.out.println();
+      System.out.println("Is the dog running:? "+ littleDog.isrunning());
 
-       System.out.println("Is the dog barking:? " + littleDog.isbarking());
+      System.out.println();
 
-       System.out.println();
+      System.out.println("is the dog sleeping:? " + littleDog.isSleeping());
 
-       System.out.println("Is the dog eating:? " + littleDog.iseating());
+      System.out.println();
 
-       System.out.println();
+      System.out.println("Is the dog barking:? " + littleDog.isbarking());
+
+      System.out.println();
+
+      System.out.println("Is the dog eating:? " + littleDog.iseating());
+
+      System.out.println();
+
+      System.out.println("The dog name is:  " + littleDog.name);
+
+  }
 
-       System.out.println("The dog name is:  " + littleDog.name);
-   }
 }
+
 ```
 
 Now you see, we’ve created a new dog with the name Bingo, you can go on to create as many dogs with different states or attributes.
 
-
 Here is the output
```

Comment:
> Here is the output
> 
> Dog running
> 
> Is the dog running:? true
> 
> Sleeping
> 
> is the dog sleeping:? true
> 
> Dog barking...
> 
> Is the dog barking:? true
> 
> Eating
> 
> Is the dog eating:? true
> 
> The dog name is: Bingo

I would change this part like so:

> Here is the output:
> 
> > Dog running
> > 
> > Is the dog running:? true
> > 
> > Sleeping
> > 
> > is the dog sleeping:? true
> > 
> > Dog barking...
> > 
> > Is the dog barking:? true
> > 
> > Eating
> > 
> > Is the dog eating:? true
> > 
> > The dog name is: Bingo

I would put a colon after *Here is the output* to be more precise. As for the output itself, It's better to make it a block quote to properly separate it from the text of your article.